src/components/TermsAndConditions/TermsAndConditions.tsx

```
@@ -0,0 +1,66 @@
+          }
+          label={
+            <Typography variant="body2">
+              <Trans
+                key="terms-link"
+                i18nKey="termsAndConditionsHtml"
+                components={[
+                  <Link
+                    href="https://ramp.bando.cool/terms"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)